packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/XDD/Document.java

```diff
@@ -0,0 +1,60 @@
+package software.uncharted.terarium.hmiserver.models.xdd;
+
+import java.util.HashMap;
+import java.util.Hashtable;
+import java.util.List;
+
+import javax.json.bind.annotation.JsonbProperty;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+
+/**
+ * XDD Document representation
+ */
+public class Document {
+
+	@JsonbProperty("_gddid")
+	public String _gddid;
+
+	@JsonbProperty("title")
+  public String title;
+
+	// FIXME: can we have the field name without the "_" prefix?
+	@JsonbProperty("_abstract")
+	@JsonAlias("abstract")
+	public String _abstract;
```

Comment:
variables should follow the regular expression '^[a-z][a-zA-Z0-9]*$'. please change to `abstract`

Reply:
this one is something I wanted to do and in fact the original field name in xdd is "abstract" but since it is a reserved keyword in java, I am not sure how to rename it without the underscore!

Reply:
Another issue with renaming fields while returning raw XDD responses from the proxy is that the actual data coming back from XDD still have bad field names. For example, a document extraction has a mapping that indicates the data will be available in a field named "askemClass" and such mapping is respected by the front-end. However, actual XDD results, which are returned to the front-end as a Response without any mapping applied, will have the bad field name such as ASKEM_CLASS.
I've added code at the front-end to remap fields to mitigate the side effect of ignoring the proper mapping, but it makes the code at the front end a little fragile. I would suggest re-visiting this issue later as time permits!

Reply:
done